### ISSUE TITLE
Longer timeout possible?

### ISSUE THREAD

**Issue by rmerom** [ConsenSys/MultiSigWallet#59]
When sending a tx from the ledger, it takes the ledger quite a while to show all the tx information.
Would it be possible to increase the UI timeout? it's pretty impatient as of right now :)

**Comment by denisgranha** [comment]
Right, this is an improvement we would like to add, not sure we can, it depends whether the timeout is raised by the [u2f API](https://github.com/google/u2f-ref-code) or by the ledger wallet.
I expect the 1st.

**Comment by rmerom** [comment]
Thanks, let me know if I can help figure it out (I can open a Dev console on the browser etc.)

**Comment by denisgranha** [comment]
basically there are two libraries involved @rmerom :
https://github.com/mastahyeti/u2f-api/blob/master/u2f-api-polyfill.js
https://github.com/LedgerHQ/ledger-node-js-api

It must be somewhere ^^
